integration_tests/src/main/python/regexp_test.py

```diff
@@ -1032,7 +1034,8 @@ def test_regexp_replace_multi_optimization(regexp_enabled):
             'REGEXP_REPLACE(a, "aa|bb|cc|dd", "PROD")',
             'REGEXP_REPLACE(a, "(aa|bb)|(cc|dd)", "PROD")',
             'REGEXP_REPLACE(a, "aa|bb|cc|dd|ee", "PROD")',
-            'REGEXP_REPLACE(a, "aa|bb|cc|dd|ee|ff", "PROD")'
+            'REGEXP_REPLACE(a, "aa|bb|cc|dd|ee|ff", "PROD")',
+            'REGEXP_REPLACE(a, "a\n|b\a|c\t", "PROD")'
```

Comment:
`gen = mk_str_gen('[abcdef\t]{0,3}')`  

We can include some of the supported characters in the input sample and also consider regex patterns like `a\ta|b\nb`, where the supported characters aren't exclusively at the end.

Reply:
done.

Reply:
Thanks! I think we may also need to change `gen` to  something like `gen = mk_str_gen('[abcdef\t]{0,3}')` to include supported characters in the input.

Reply:
oh I missed that, thanks, updated.